cumulus/pallets/collator-selection/src/benchmarking.rs

```diff
@@ -394,7 +394,7 @@ mod benchmarks {
 		register_validators::<T>(c);
 		register_candidates::<T>(c);
 
-		let new_block: BlockNumberFor<T> = 1800u32.into();
+		let new_block: BlockNumberFor<T> = 3600u32.into();
```

Comment:
Not your fault as the benchmark before was not correct. However, changing this will break the benchmark for anyone downstream that doesn't change the slot duration to 6 seconds. This needs to made generic.

Reply:
Working on a fix.

Reply:
Fix [here](https://github.com/paritytech/polkadot-sdk/pull/3189).